classes/global.html

```diff
@@ -7,19 +7,20 @@
 Code distributed by Google as part of the polymer project is also
 subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
 -->
+
 <link rel="import" href="../paper-styles-classes.html">
 
 <!--
-A set of base styles that are applied to the document and standard elements that
-match the Material Design spec.
+Note that this file probably doesn't do what you expect it to do. It's not
+a `<style is=custom-style include="..."` type of style include, which mean
+these styles will only apply to the main document, regardless of where
+you import this file.
+
+For a set of styles that can be applied to an element,
+check iron-demo-helpers/demo-pages-shared-styles.html.
```

Comment:
I totally agree that this part is good:

> Note that this file probably doesn't do what you expect it to do. It's not
a `<style is=custom-style include="..."` type of style include, which mean
these styles will only apply to the main document, regardless of where
you import this file.

But in this part:

> For a set of styles that can be applied to an element,
check iron-demo-helpers/demo-pages-shared-styles.html.

the referenced page (https://github.com/PolymerElements/iron-demo-helpers/blob/2.0-preview/demo-pages-shared-styles.html) doesn't seem related to this file, AFAICT. Maybe these files are roughly the same thing and I'm missing it because they're organized differently?